src/FixedProductMarketMakerMapping.ts

```diff
@@ -124,11 +130,14 @@ function recordFPMMLiquidity(fpmm: FixedProductMarketMaker,
     let fpmmTransaction = new FpmmTransaction(fpmmLiquidityId);
     fpmmTransaction.fpmm = fpmm.id;
     fpmmTransaction.user = funder;
+    fpmmTransaction.fpmmType = FPMM_TYPE_LIQUIDITY;
     fpmmTransaction.transactionType = liquidityType;
     fpmmTransaction.collateralAmount = sharesAmount;
+    fpmmTransaction.collateralAmountUSD = sharesAmountUSD;
```

Comment:
This is consistent with the existing behavior, but I just realized that what is being assigned as the `collateralAmount` in this case is the `sharesAmount`, which is *not* denominated in terms of collateral, but rather something a bit more arbitrary. I personally would have based this off of the `fpmmLiquidity.additionalLiquidityParameter` value, but alas, I didn't comment on that earlier, so I'm making my peace with this now.